package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "linear-zapier",
-  "version": "4.4.5",
+  "version": "4.4.6",
```

Comment:
Just bump this as needed before merging, since I had to deploy a bug fix and user this version already

Reply:
sounds good. i will check the version, when we are ready to merge. dont want to do it just now, to not release this accidentally